README.md

```diff
@@ -30,6 +30,11 @@ lein doc
 ## Releases
 
 ```clojure
+# Patch version releases (minor and major increments require a manual edit of version).
+# Expect this task to modify project files and deploy to Clojars.
+# See `release-tasks` in project.clj for details.
+lein release
```

Comment:
This is already recommended a few lines down. Could you consolidate them and separate the instructions so it doesn't look like you should execute them blindly top-to-bottom? I'm often in a hurry when reading this section.

Reply:
Yeah, I'll work on this tomorrow morning when my brain is fresh. I think there is more to do here too, such as warn about the mistakes I made when doing my release 😃 .